tests/ops/api/v1/endpoints/test_dataset_endpoints.py

```diff
@@ -140,6 +140,7 @@ def test_put_validate_dataset_nested_collections(
             validate_dataset_url, headers=auth_header, json=invalid_dataset
         )
         json_response = json.loads(response.text)
+        print(json_response)
```

Comment:
We'll want to remove the `print`s in this file I think

Reply:
@seanpreston i left these in specifically. Prints are very helpful in tests, especially with complicated object comparisons. They also don't actually get printed unless that specific test fails, so it also doesn't hurt anything to leave it in